components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -491,9 +493,18 @@ bool AdsServiceImpl::IsEnabled() const {
 }
 
 uint64_t AdsServiceImpl::GetAdsPerHour() const {
+  uint64_t ads_per_hour = GetUint64Pref(ads::prefs::kAdsPerHour);
+  if (ads_per_hour == 0) {
+    const base::Feature kAdServing{"AdServing",
```

Comment:
features should not be defined this way. See partial fix here and comment for actual fix https://github.com/brave/brave-core/pull/11223/commits/00a9face5b5f4be1bfccc1a788dc607715237b6a#r789168565

Reply:
Thanks I will raise an issue and get these resolved

Reply:
@bridiver I have checked and there are no more occurrences, we will make sure to follow these guidelines. Thanks